pkg/mcs/resourcemanager/server/config.go

```diff
@@ -108,7 +108,8 @@ type ControllerConfig struct {
 }
 
 // Adjust adjusts the configuration and initializes it with the default value if necessary.
-func (rmc *ControllerConfig) Adjust(meta *configutil.ConfigMetaData) {
+// FIXME: is it expected?
+func (rmc *ControllerConfig) Adjust(_ *configutil.ConfigMetaData) {
```

Comment:
`Adjust` doesn't use meta, is it expected?

Reply:
IMO, it's better use `meta` for the method `Adjust` of `RequestUnitConfig`.